src/features/articles/single-article/singleArticleReducer.js

```diff
@@ -6,7 +6,7 @@ const singleArticleReducer = (state, { type, article, errors }) => {
       return {
         ...state,
         currentArticle: article,
-        errors: undefined,
+        errors: {},
```

Comment:
@chukwuemekachm is there a particular reason for this? Returning undefined removes it from the state and makes it possible to do checks like if(!errors) ..., You may want to return it to its original stage as there is no simple way of checking if it is empty if it is there.

Reply:
Thank you @Veraclins for your awesome review. I've implemented all your feedback except this one. Reason being that when you remove it from the state while a component is still subscribed to it, it spills `prop-types` validation errors on the console. Because at the time when it is updated to undefined the `errors` will no longer be present in the store.
If you are worried as to how you can check it try if(!errors.tittle). This is cool because you are being specific on the error you are checking for.
Once again, thanks for the feedback.